ocdskingfisherarchive/database_process.py

```diff
@@ -0,0 +1,31 @@
+import sqlalchemy as sa
+from ocdskingfisherarchive.collection import Collection
+
+
+class DataBaseProcess:
```

Comment:
I thought we had decided to just use the filesystem, not the database.

The database will contain loads of collections that no longer have any files and/or the filesystem can contain files that are no longer loaded.

Reply:
I had the SQL Query in the Google Doc?

I think it is easier to pick up, in both cases ( a collection from Scrapy and a collection from a Local Load). In the later case, we'll need something to check to make sure the collect stage has finished, and if we can't check a Scrapy log we'll need to check something.

> The database will contain loads of collections that no longer have any files 

That will be detected and put into local storage, so next time it runs it will skip that collection quickly.

> the filesystem can contain files that are no longer loaded.

That's another case I'd like to look at after cracking the database case, as I can see some edge cases here.